sequencer/coordinator/prover/prover.go

```diff
@@ -70,6 +77,24 @@ type Status struct {
 	PubData string     `json:"pubData"`
 }
 
+// IsReady returns true when the prover is ready
+func (status StatusCode) IsReady() bool {
+	if status == StatusCodeAborted || status == StatusCodeFailed || status == StatusCodeSuccess ||
+		status == StatusCodeUnverified || status == StatusCodeReady {
+		return true
+	}
+	return false
+}
+
+// IsInitialized returns true when the prover is initialized
+func (status StatusCode) IsInitialized() bool {
```

Comment:
What is prover initialization?

Reply:
@kaiden-araki  Prover gets initialised in node.go, here we need to initialise single prover instead of an array. https://github.com/tokamak-network/tokamak-sybil-resistance-mvp/blob/develop/sequencer/node/node.go#L428